default.nix

```diff
@@ -330,8 +324,44 @@ let
           rm pre-stylish.diff post-stylish.diff
           exit
         '';
+
+        updateClientDeps = pkgs.writeScript "update-client-deps" ''
+          if [ ! -f package.json ]
+          then
+              echo "package.json not found. Please run this script from the client directory." >&2
+              exit 1
+          fi
+
+          echo Installing JavaScript Dependencies
+          ${pkgs.yarn}/bin/yarn
+          echo Generating psc-package config
+          ${pkgs.yarn}/bin/yarn spago psc-package-insdhall
+          echo Installing PureScript Dependencies
+          ${pkgs.yarn}/bin/yarn psc-package install
+          echo Generating nix config
+          ${pp2n}/bin/pp2n psc-package2nix
+          ${yarn2nix.yarn2nix}/bin/yarn2nix > yarn.nix
+          cp .psc-package/local/.set/packages.json packages.json
+          echo Done
+        '';
+      };
+
+      pp2nSrc = pkgs.fetchFromGitHub {
```

Comment:
Don't put these as elements of `scripts` otherwise `release.nix` will try to build them.

Reply:
Oh wait, they're not, ignore me.